src/utils/graph_mutations.js

```diff
@@ -1,18 +1,25 @@
-import { gql } from '@apollo/client';
+import { gql } from "@apollo/client"
 
 export const SUBMIT_MOOD = gql`
-mutation createMood($mood: Int!, $description: String) {
-  createMood(
-    input: {
-      params: {
-        mood: $mood,
-        description: $description
+  mutation createMood($mood: Int!, $description: String) {
+    createMood(input: { params: { mood: $mood, description: $description } }) {
+      user {
+        id
       }
     }
-  ) {
-    user {
-      id
+  }
+`
+//mfer this one's input type got me ;/ aaahhhhhhh!
```

Comment:
Don't forget to delete comments :)

Reply:
hahaha just to express my anger. Takes forever to figure that out

Reply:
Comment deleted